### PR TITLE
Fix the user guide format

### PR DESCRIPTION
**Pull request by lancehaha**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -156,9 +156,9 @@ Examples:
...
 ### Finding a person: `find`
 
-Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
+Format: `find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tag`
 
-* any possible orders and combinations of the 3 parameters name, phone number and room number are applicable
+* any possible orders and combinations of the 4 parameters NAME, PHONE_NUMBER, ROOM_NUMBER, and TAG are applicable
 
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
```
Comment (EdaYeo) [inline docs/UserGuide.md:164]: Currently the search isn't  case insensitive for tags, should we write that into the UG or fix it for the tags?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -167,14 +167,14 @@ Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * Only full words will be matched for room_number and phone_number, the arguments
```
Comment (EdaYeo) [inline docs/UserGuide.md:169]: Should it be ROOM_NUMBER and PHONE_NUMBER here too?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -167,14 +167,14 @@ Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
 * Persons matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (EdaYeo) [inline docs/UserGuide.md:167]: The `OR` search is only implemented for names and not for tags. Should this be written into the UG, or fixed within the tags?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -167,14 +167,14 @@ Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
...
-  should be exactly the same as the person's room number and phone number
-* order of the tags doesn't matter `t/friends colleagues` works the same
+  should be exactly the same as the person's ROOM_NUMBER and PHONE_NUMBER
+* Order of the tags do not matter `t/friends colleagues` works the same
   as `t/colleagues friends`
 
 Examples:
-* `find p/test_number n/test_name r/08-1234 t/friends colleagues`  
+* `find p/94509592 n/Alex Jones r/08-1234 t/friends colleagues`  
   finds people whose both your friend and colleagues.
```
Comment (EdaYeo) [inline docs/UserGuide.md:177]: Should "whose" be changed to "who are"?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -167,14 +167,14 @@ Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
...
 * Only full words will be matched for room_number and phone_number, the arguments
-  should be exactly the same as the person's room number and phone number
-* order of the tags doesn't matter `t/friends colleagues` works the same
+  should be exactly the same as the person's ROOM_NUMBER and PHONE_NUMBER
+* Order of the tags do not matter `t/friends colleagues` works the same
   as `t/colleagues friends`
 
 Examples:
-* `find p/test_number n/test_name r/08-1234 t/friends colleagues`  
+* `find p/94509592 n/Alex Jones r/08-1234 t/friends colleagues`
```
Comment (EdaYeo) [inline docs/UserGuide.md:176]: The find here for t/friends colleagues only works if the two words are in one tag as "friends colleagues" but doesn't work if the person has two tags, "friends" and "colleagues". Should this be written into the UG or fixed for tags?

**Review by EdaYeo — COMMENTED** [review]
LGTM, just some nits to fix!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ the process of accessing and updating resident student details. What's more, Dor
     - [Adding a person : `add`](#adding-a-person-add)
     - [Listing all persons : `list`](#listing-all-persons--list)
     - [Editing a person : `edit`](#editing-a-person--edit)
-    - [Finding a person : `find`](#locating-persons-by-name-phone_number-room_number-or-tag-find)
+    - [Finding a person : `find`](#locating-persons-by-name-phone-room_number-or-tag-find)
```
Comment (EdaYeo) [inline docs/UserGuide.md:32]: This leads to a broken hyperlink. Should it (#finding-a-person--find) instead? Or should we update the title of find?

File: docs/UserGuide.md [hunk 7]
```diff
@@ -156,25 +156,27 @@ Examples:
 
 ### Finding a person: `find`
 
-Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/TAG’
+Format: `find n/NAME p/PHONE_NUMBER  r/ROOM_NUMBER t/Tag`
```
Comment (EdaYeo) [inline docs/UserGuide.md:159]: Should this be t/TAG instead?

File: docs/UserGuide.md [hunk 8]
```diff
@@ -289,11 +291,11 @@ _Details coming soon ..._
 
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**Add**    | `add [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add**    | `add [n/NAME] [p/PHONE_NUMBER ] [e/EMAIL] [a/ADDRESS] [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
```
Comment (EdaYeo) [inline docs/UserGuide.md:294]: Should the additional white space at the end in [p/PHONE_NUMBER ] be removed? The same goes for Edit and Find below.

**Review by EdaYeo — COMMENTED** [review]
A few more comments, after that should be good to merge.

File: docs/UserGuide.md [hunk 9]
```diff
@@ -130,14 +130,14 @@ Format: `add n/NAME p/PHONE e/EMAIL [r/ROOM_NUMBER] [a/ADDRESS] [t/TAG]...`
...
 > * `EMAIL` should be of the format local-part@domain
 > * You cannot set emergency contact details when adding a person. Use the `edit` command to add emergency contact details.
 > * You cannot set graduation year when adding a person. Use the `edit` command to add graduation year.
 > * Refer to [Field constraints](#field-constraints) for more details on accepted values for each field.
 
 > <span style="color:Tomato"> WARNING! </span> <br>
-> 
+>
 > * If there are duplicate names, i.e if a person in the DorManagerPro address book already has the specified `NAME`, an error will be thrown. This is because it is very rare for two people to have the exact same name down to the surname. Instead, we allow numerals to denote different people with the same name.
 > * If there are duplicate phone numbers, i.e if a person in the DorManagerPro address book already has the specified `PHONE_NUMBER`, an error will be thrown. This is because no two people have the same phone number.
```
Comment (chashaobao123) [inline docs/UserGuide.md:142]: PHONE

**Review by SeanFoongjt — APPROVED** [review]
LGTM!

**Pull request merged**